### PR TITLE
add ass 3

### PR DESCRIPTION
**Pull request by Erdubya**
draw a dick

### CODE REVIEW

**Comment by alex-g-carpenter** [comment]
that's some ass for sure

File: php-lover/assignment3.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    sköldpadda.home()
+    sköldpadda.circle(r)
+    sköldpadda.left(180)
+    sköldpadda.circle(r)
+
+length = 100
+ball_size = 30
+
+screen = turtle.getscreen()
+sköldpadda = turtle.Turtle()
```
Comment (alex-g-carpenter) [inline php-lover/assignment3.py:23]: bruh

Comment (Erdubya) [reply php-lover/assignment3.py]: :crab:

**Review by colinrlly — APPROVED** [review]
Very nice use of skoldpadda

**Comment by Erdubya** [comment]
> Very nice use of skoldpadda

tack

**Pull request merged**